sci-electronics/fritzing/fritzing-0.9.4.ebuild

```
@@ -0,0 +1,94 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg
+
+MY_PV="CD-498"
+PARTS_P="${PN}-parts-${PV}"
+PARTS_COMMIT="e79a69765026f3fda8aab1b3e7a4952c28047a62"
+DESCRIPTION="Electronic Design Automation"
+HOMEPAGE="https://fritzing.org/"
+SRC_URI="https://github.com/fritzing/fritzing-app/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/fritzing/fritzing-parts/archive/${PARTS_COMMIT}.zip -> ${PARTS_P}.zip"
+
+LICENSE="CC-BY-SA-3.0 GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+PLOCALES=( ar bg bn cs da de el en es et fa fr gl hi hu id it ja ko mk mr nl pl pt-BR pt-PT ro ru sk sl sr sv th tr uk ur vi zh-CN zh-TW )
+IUSE="${PLOCALES[@]/#/l10n_}"
+
+REQUIRED_USE="|| ( ${IUSE[@]} )"
+
+RDEPEND="dev-qt/qtconcurrent:5=
+	dev-qt/qtcore:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtnetwork:5=
+	dev-qt/qtprintsupport:5=
+	dev-qt/qtserialport:5=
+	dev-qt/qtsql:5=[sqlite]
+	dev-qt/qtsvg:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtxml:5=
+	>=dev-libs/quazip-0.7.2[qt5(+)]
+	dev-libs/libgit2:=
+"
+DEPEND="
+	${RDEPEND}
+	>=dev-libs/boost-1.40
+"
+
+S="${WORKDIR}/${PN}-app-${MY_PV}"
+
+DOCS="README.md"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-libgit2-version.patch"
+	"${FILESDIR}/${P}-move-parts-db-path.patch"
+)
+
+src_prepare() {
+	local translations=
+
+	# fix build with newer quazip - bug #597988
+	sed -i -e "s/#include <quazip/&5/" src/utils/folderutils.cpp || die
+	sed -i -e "s|/usr/include/quazip|&5|" -e "s/-lquazip/&5/" phoenix.pro || die
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -rf src/lib/quazip/ pri/quazip.pri src/lib/boost* || die
```

We avoid using the `-f` option since it hides errors in the ebuild.